src/fake-timers-src.js

```diff
@@ -188,26 +186,6 @@ function withGlobal(_global) {
         _global.setImmediate && typeof _global.setImmediate === "function";
     const intlPresent = _global.Intl && typeof _global.Intl === "object";
 
-    // Make properties writable in IE, as per
-    // https://www.adequatelygood.com/Replacing-setTimeout-Globally.html
-    /* eslint-disable no-self-assign */
-    if (isRunningInIE) {
-        _global.setTimeout = _global.setTimeout;
-        _global.clearTimeout = _global.clearTimeout;
-        _global.setInterval = _global.setInterval;
-        _global.clearInterval = _global.clearInterval;
-        _global.Date = _global.Date;
-    }
-
-    // setImmediate is not a standard function
-    // avoid adding the prop to the window object if not present
-    if (setImmediatePresent) {
-        _global.setImmediate = _global.setImmediate;
```

Comment:
Node actually has setImmediate as well

Reply:
Yes, not removing that. Just the IE specific x=x thingie.